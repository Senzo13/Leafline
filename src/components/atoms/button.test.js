import React from "react";
import TestRenderer from "react-test-renderer";
import Button from "./button"; // changez cela pour votre composant

describe("MyComponent", () => {
  it("renders correctly", () => {
    const tree = TestRenderer.create(<Button />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
