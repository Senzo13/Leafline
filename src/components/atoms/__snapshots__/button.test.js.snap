// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
<View>
  <Text
    style={
      {
        "color": "black",
        "fontWeight": "bold",
      }
    }
  >
    Button
  </Text>
</View>
`;
